articles/integrating-b2c-mpesa-api-using-node-js/index.md

```diff
@@ -0,0 +1,299 @@
+### Integrating B2C Mpesa API using Node.js
+
+Mpesa can be described as a phone-based mobile banking service. Since its inauguration in 2007 by Vodafone Group PLC and Safaricom in Kenya, it has expanded to eight more countries in Africa.
```

Comment:
Change `M-Pesa` spelling throughout the paper

Reply:
done.